modules/ROOT/pages/database-administration/standard-databases/manage-databases.adoc

```diff
@@ -803,7 +806,8 @@ DROP DATABASE customers IF EXISTS
 The `DROP DATABASE` command will remove a database entirely.
```

Comment:
I think this comment should mov up above the new subsection for `IF EXISTS` 🤔

Reply:
Let me move it.